### DISCUSSION TITLE
the procedure entry point nghttp2_option_set_no_rfc9113_leading_and_trailing_ws_validation could not be located in the dynamic link libarary C:\laragon\bin\php\php-8.3.1\etx\php_curl.dlll

### DISCUSSION THREAD

**Discussion by salmanelzaydy** [leokhoa/laragon#651] [General]
with PHP 8.2 or 8.3 I get this error 

the procedure entry point nghttp2_option_set_no_rfc9113_leading_and_trailing_ws_validation could not be located in the dynamic link libarary C:\laragon\bin\php\php-8.3.1\etx\php_curl.dlll

I can't solve it and I have a problem with curl ext

**Comment by dustinatx** [comment]
Update Apache to the latest version.

https://www.kreaweb.be/laragon-update-apache/

**Reply by mehediishere** [reply]
@dustinatx  Yup, that helps. Thank you

**Reply by justemalachie** [reply]
Thank you so much.
